src/MainContent.js

```diff
@@ -9,23 +9,35 @@ import Profile from './containers/profile/ProfileContainer';
 import Leaderboard from './containers/leaderboard/Leaderboard';
 import './MainContent.css';
 import { Col } from 'reactstrap';
+window.scrollTo(0, 0);
```

Comment:
This code should be in index.js.

Reply:
DidUpdate do not work in app.js after route change.
So this is the highest possible level of the overall component.